openmls/src/group/mls_group/test_mls_group.rs

```diff
@@ -269,14 +268,10 @@ fn remover(ciphersuite: &'static Ciphersuite, backend: &impl OpenMlsCryptoProvid
         }
 
         // Check that Alice removed Bob
-        // TODO #541: Replace this with the adequate API call
-        assert_eq!(
-            staged_proposal
-                .sender()
-                .as_key_package_ref()
-                .expect("An unexpected error occurred."),
-            alice_kpr
-        );
+        assert!(matches!(
+            staged_proposal.sender(),
+            Sender::Member(member) if member == alice_kpr
```

Comment:
I think I've never seen this construction in rust before. What is going on here?

Reply:
This is specific to the `matches!` macro: https://doc.rust-lang.org/std/macro.matches.html

Reply:
Very cool! I had no idea this was a possibility. Seems useful, though.